src/test/java/de/doubleslash/usb_led_matrix/view/ConfigurationViewTest.java

```diff
@@ -124,39 +125,58 @@ void shouldLeaveTimeoutWhenColorChanges() {
    @Test
    void shouldTurnOffWhenCurrentTimeEqualsTimeout() {
       // Given
-      final LocalTime currentTime = LocalTime.of(21, 0);
+      final LocalTime currentTime = LocalTime.of(20, 0);
+
       configurationView.setUsbAdapter(usbAdapter);
       CommandLineOptions.setTimeoutInMinutes(60);
-      configurationView.timeLightsTurnedOffNow = LocalTime.of(20, 0);
+      configurationView.timeLightsTurnedOffNow = LocalTime.of(21, 0);
       // When
-      configurationView.turnOffAutomaticallyIfNeeded(currentTime);
+      configurationView.turnOffAutomaticallyIfNeeded(currentTime, LocalDate.now());
       // Then
-      verify(usbAdapter).updatePixel(Color.BLACK);
+      verify(usbAdapter, never()).updatePixel(Color.BLACK);
```

Comment:
- You did change an existing test (is the previoius test case not valid anymore?)
- test name does not reflect what the test is testing

Reply:
Done

Reply:
I do not see any change done here?

Reply:
I have changed the name